### PR TITLE
feat: Update Swift version for `Domain` and `Core`

### PR DESCRIPTION
**Pull request by RaniaZeid**

### CODE REVIEW

File: Dokan/Dokan/Scenes/Verification/VerificationViewController.swift [hunk 1]
```diff
@@ -35,10 +35,20 @@ class VerificationViewController: UIViewController {
             // TODO: - Handle the textField upon being filled
```
Comment (ahmdmhasn) [inline Dokan/Dokan/Scenes/Verification/VerificationViewController.swift]: To be ignored.

File: Dokan/Dokan/Scenes/Verification/VerificationViewController.xib [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (ahmdmhasn) [inline Dokan/Dokan/Scenes/Verification/VerificationViewController.xib:1]: To be ignored.

File: UIDokan/Sources/UIDokan/ReusableViews/OTCTextField.swift [hunk 3]
```diff
@@ -7,9 +7,9 @@
```
Comment (ahmdmhasn) [inline UIDokan/Sources/UIDokan/ReusableViews/OTCTextField.swift]: To be ignored.

**Review by ahmdmhasn — CHANGES_REQUESTED** [review]
Great effort ya @RaniaZeid. Please note that the pull request should focus on a single change and we should exclude any other unrelated changes.

**Comment by ahmdyasser** [comment]
@RaniaZeid resolve the conversations to merge this PR

**Review by ahmdmhasn — APPROVED** [review]
🚀

**Review by ahmednaagy — APPROVED** [review]

**Review by claramounir — APPROVED** [review]

**Review by RaniaZeid — COMMENTED** [review]
resolved

**Pull request merged**